www/js/lib/uiUtil.js

```diff
@@ -862,15 +862,12 @@ function applyAppTheme (theme) {
 
 // Displays the return link and handles click event. Called by applyAppTheme()
 function showReturnLink () {
-    var viewArticle = document.getElementById('viewArticle');
```

Comment:
Since you have more than one `document.getElementById('viewArticle')` statements below, it would be much better to define a variable here (like it used to do). Obviously you need t give it a different name, like `viewArticleLink`, to avoid a clash with the `viewArticle()` function. Alternatively, you could rename the viewArticle function to `showArticle()`. It's up to you, whatever is easiest.